packages/ui-docs/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@blockstack/ui-docs",
   "version": "1.0.3",
   "dependencies": {
-    "@blockstack/ui": "^2.9.5",
+    "@blockstack/ui": "^2.9.7",
```

Comment:
Please use beta versions whenever publishing outside of CI - otherwise it breaks our auto-publish stuff: https://github.com/blockstack/ux/runs/875078456?check_suite_focus=true

Reply:
Good to know